content/tutorials/manage-the-rippled-server/install-rippled.md

```diff
@@ -46,59 +85,234 @@ This section assumes that you are using CentOS 7 or Red Hat Enterprise Linux 7.
+
+    $ sudo service rippled restart
+
+### Connect Your `rippled` to the XRP Test Net
+
+Ripple has created the [XRP Test Network](https://ripple.com/build/xrp-test-net/) to provide a testing platform for the XRP Ledger. XRP Test Net funds are not real funds and are intended for testing only. You may want to connect your `rippled` server to the XRP Test Net to test out and understand `rippled` functionality before connecting to the production XRP Ledger Network. ***TODO: stated correctly?***
+
+**Note:** The XRP Test Net ledger and balances are reset on a regular basis.
+
+_**To connect your `rippled` server to the XRP Test Net, set the following configurations:**_
+
+1. In your `rippled.cfg` file:
+
+      a. Uncomment the following section, as follows:
+
+        [ips]
+        r.altnet.rippletest.net 51235
+
+      b. Comment out the following section, as follows:
 
-You can use the [rippled commandline interface](get-started-with-the-rippled-api.html#commandline) as follows:
+        # [ips]
+        # r.ripple.com 51235
 
-    $ /opt/ripple/bin/rippled <METHOD>
+2. In your `validators.txt` file:
 
-For a full list of available methods, see the [rippled API reference](rippled-api.html).
+      a. Uncomment the following sections, as follows:
 
-### Additional Configuration
+        [validator_list_sites]
+        https://vl.altnet.rippletest.net
 
-`rippled` should connect to the XRP Ledger with the default configuration. However, you can change your settings by editing the `rippled.cfg` file (located at `/opt/ripple/etc/rippled.cfg` when installing `rippled` with yum). For recommendations about configuration settings, see [Capacity Planning](capacity-planning.html).
+        [validator_list_keys]
+        ED264807102805220DA0F312E71FC2C69E1552C9C5790F6C25E3729DEB573D5860
 
-See [the `rippled` GitHub repository](https://github.com/ripple/rippled/blob/master/cfg/rippled-example.cfg) for a description of all configuration options.
+      b. Comment out the following sections, as follows:
 
-Changes to the `[debug_logfile]` or `[database_path]` sections may require you to give the `rippled` user and group ownership to your new configured path:
+        # [validator_list_sites]
+        # https://vl.ripple.com
+        #
+        # [validator_list_keys]
+        # ED2677ABFFD1B33AC6FBC3062B71F1E8397C1505E1C42C64D11AD1B28FF73F4734
 
-        $ chown -R rippled:rippled <configured path>
+3. Restart `rippled`.
 
-Restart `rippled` for any configuration changes to take effect:
+4. To verify that your `rippled` is connected to the XRP Test Net, go to the [XRP Test Net Faucet](https://developers.ripple.com/xrp-test-net-faucet.html) and click **Generate credentials**. Make a note of the **Address** value. Make the [`account_info`](https://developers.ripple.com/account_info.html) request to your `rippled` server. ***TODO: Any ideas for a better way to this check?***
```

Comment:
This works. You can also run `server_info` and see what ledger version your server is on (assuming the state is `full`/`proposing` and it's not still syncing). If it's close to or the same as what `s1.ripple.com`'s server_info reports, then you're on the production network. If it's much smaller (like, fewer digits) then you're on the testnet.